packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.js

```diff
@@ -52,75 +57,111 @@ function QuickTalk ({
   const [_expand, setExpand] = React.useState(expand)
   const badge = (showBadge && comments.length > 0) ? comments.length : false
 
-  const QTButton = (fixedPosition) ? FixedButton : Button
+  const QTButtonContainer = (fixedPosition) ? FixedButtonContainer : ButtonContainer
   const QTPanel = (fixedPosition) ? FixedBox : Box
 
   if (!_expand) {
     return (
-      <QTButton
-        a11yTitle={t('QuickTalk.aria.openButton', { count: comments.length })}
-        onClick={() => setExpand(true)}
-        icon={<Chat />}
-        badge={badge}
-      />
+      <QTButtonContainer
+        background={{ dark: 'dark-3', light: 'light-3' }}
+        elevation='small'
+      >
+        <Button
+          a11yTitle={t('QuickTalk.aria.openButton', { count: comments.length })}
+          onClick={() => setExpand(true)}
+          icon={<Chat />}
+          badge={badge}
+        />
+      </QTButtonContainer>
     )
   }
 
   return (
-    <QTPanel
-      a11yTitle={t('QuickTalk.aria.mainPanel')}
-      elevation='medium'
-      role='dialog'
-      background={{ dark: 'dark-3', light: 'light-3' }}
-    >
-      <Box
-        direction='row'
-        flex={false}
-        justify='between'
-        pad='small'
+    <Keyboard onEsc={() => setExpand(false)}>
+      <QTPanel
+        a11yTitle={t('QuickTalk.aria.mainPanel')}
+        elevation='medium'
+        role='dialog'
+        background={{ dark: 'dark-3', light: 'light-3' }}
       >
-        <Anchor
-          a11yTitle={t('QuickTalk.aria.goToTalk')}
-          label={t('QuickTalk.headerLink')}
-          href={subject.talkURL}
-          target='_blank'
-          icon={<Chat />}
-        />
-        <Button
-          a11yTitle={t('QuickTalk.aria.closeButton')}
-          autoFocus={true}
-          icon={<Close size='small' />}
-          onClick={() => setExpand(false)}
-          plain
-        />
-      </Box>
-      <Box overflow='auto'>
-        <Box flex={false} pad={{ bottom: 'small', left: 'small', right: 'small' }}>
-          <UnorderedList as='ul' flex={false}>
-            {comments.map(comment => {
-              const author = authors[comment.user_id]
-              const roles = authorRoles[comment.user_id]
+        <Box
+          direction='row'
+          flex={false}
+          justify='between'
+          pad='small'
+        >
+          <Anchor
+            a11yTitle={t('QuickTalk.aria.goToTalk')}
+            label={t('QuickTalk.headerLink')}
+            href={subject.talkURL}
+            target='_blank'
+            icon={<Chat />}
+          />
+          <Button
+            a11yTitle={t('QuickTalk.aria.closeButton')}
+            autoFocus={true}
```

Comment:
Here's the meandering & confused tale of how I got here:

- I designed the QuickTalk as a non-modal dialog. (Why I didn't go for modal is a separate tale.)
- I tried adding `role=dialog` thinking _"ahhah, it's a separate content from the rest of the UI, so role=dialog should help people understand this component!"_
- You helpfully [pointed out](https://github.com/zooniverse/front-end-monorepo/pull/3048#pullrequestreview-971240072) that a dialog actually needs additional behaviour, namely (1) the dialog should receive focus when it opens, and (2) that focus should go to the [first comment](https://github.com/zooniverse/front-end-monorepo/pull/3048#discussion_r871253530) (or at least top) of the dialog 
- I couldn't focus on the first comment, since there may be scenarios where the panel opens with 0 comments.
- I tried making the "Subject Discussions" header/link the first focused item, but I couldn't get the element.to.focus() since the element wasn't rendered when the "open dialog" function triggered. (NOTE: I will have to try this again now that I know a bit more about [callback refs](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs))
- I found that Grommet's `<Anchor autoFocus={true} />` doesn't work, so that's out. (Variations such as `autofocus={true}`, `autofocus`, and `autoFocus` also failed to register the intended property on the Grommet element.)
- The only element/Grommet component I could give focus to using the autofocus prop is a Button, and so...

...that's how & why the Close button got focus when the dialog pops up. As usual, there's probably a simpler solution to this that I'm overlooking, but man, this so far has been a trip.

- Side note 1: `<Box overflow='auto' tabIndex='0' autofocus={true} / autoFocus={true} / autofocus / autoFocus>`, for the content box which contains all the comments, also failed to gain focus. I really wanted this to get focus, since this is the actual _content_ of the dialog.
- Side note 2: Curiously, I see `<Box autoFocus />` used in SlideTutorial, but I think what's getting autofocus when the SlideTutorial modal appears is a hidden `<a>`nchor, not the intended Box. 🤷 Side _side_ note: I think the SlideTutorial is missing a `<Box tabindex=0>` that should allow a user to scroll through vertically-tall tutorial content.

Reply:
TBH the close button should be focused, since it’s the first focusable element in the popup. 

I think the ARIA advice for dialogs mentions that too.